agents/acu/acu_agent.py

```diff
@@ -407,25 +419,47 @@ def start_monitor(self, session, params=None):
             # more keys to any self.data['status'] categories beyond this point
             influx_status = {}
             for category in self.data['status']:
-                for statkey in self.data['status'][category].keys():
-                    if type(self.data['status'][category][statkey]) == float:
-                        if self.data['status'][category][statkey] == float('nan'):
-                            influx_status[statkey + '_influx'] = 0.0
-                        else:
+                if category != 'commands':
+                    for statkey in self.data['status'][category].keys():
+                        if type(self.data['status'][category][statkey]) == float:
+                     #   if category == 'commands':
+                     #       if self.data['status'][category][statkey] == float('nan'):
+                     #           continue
+                     #       else:
+                     #           influx_status[statkey + '_influx'] = self.data['status'][category][statkey]
+                      #  else:
                             influx_status[statkey + '_influx'] = self.data['status'][category][statkey]
-                    elif type(self.data['status'][category][statkey]) == str:
-                        if self.data['status'][category][statkey] in ['None', 'True', 'False']:
-                            influx_status[statkey + '_influx'] = tfn_key[self.data['status'][category][statkey]]
-                        else:
-                            influx_status[statkey + '_influx'] = mode_key[self.data['status'][category][statkey]]
-                    elif type(self.data['status'][category][statkey]) == int:
-                        if statkey in ['Year', 'Free_upload_positions']:
-                            influx_status[statkey + '_influx'] = float(self.data['status'][category][statkey])
+                        elif type(self.data['status'][category][statkey]) == str:
+                            if self.data['status'][category][statkey] in ['None', 'True', 'False']:
+                                influx_status[statkey + '_influx'] = tfn_key[self.data['status'][category][statkey]]
```

Comment:
This will turn 'None' into a float, but 'True' and 'False' into ints.  So I think this should be two separate "if" blocks.